avm/utilities/tools/Set-AVMModule.ps1

```diff
@@ -109,6 +109,20 @@ function Set-AVMModule {
         # load cross-references
         $crossReferencedModuleList = Get-CrossReferencedModuleList
 
+        # load AVM references (done to reduce WebRequests to GitHub repository)
+        # Telemetry
+        $telemetryUrl = 'https://aka.ms/avm/static/telemetry'
+        try {
+            $rawReponse = Invoke-WebRequest -Uri $telemetryUrl
```

Comment:
Minor typo, fix if you may, for all occurrences
```suggestion
            $rawResponse = Invoke-WebRequest -Uri $telemetryUrl
```

Reply:
Fixed via search & replace